ethsigner/src/main/java/tech/pegasys/ethsigner/requesthandler/sendtransaction/TransactionTransmitter.java

```diff
@@ -83,9 +83,8 @@ private JsonRpcBody createSignedTransactionBody() {
       return new JsonRpcBody(JsonRpcError.INTERNAL_ERROR);
     }
 
-    final JsonRpcRequest sendRawTransaction =
-        new JsonRpcRequest(
-            JSON_RPC_VERSION, JSON_RPC_METHOD, singletonList(signedTransactionHexString));
+    final JsonRpcRequest sendRawTransaction = new JsonRpcRequest(JSON_RPC_VERSION, JSON_RPC_METHOD);
+    sendRawTransaction.setParams(singletonList(signedTransactionHexString));
```

Comment:
Can we add a 2nd constructor just for our own use instead of having to use a constructor and setter here. Not sure if jackson will like that or not though.

Reply:
I thought about that ...no.

I'm thinking that constructors are best left for `final` variables. Besides, I'm not sure what happens with Jackson deserialisation (it should be fine 🙄 )

Reply:
Still think the code would be more readable with another constructor there as this is a very common case.